Round/shared.css

```diff
@@ -210,3 +211,35 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
   border-radius: 20px !important;
   overflow: hidden;
 }
+
+/* Main-Menu Guides */
+.overlayguides_GuideListItemBase_1etIu {
+  border-radius: 20px;
+}
+
+/* Main-Menu Achievements */
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_AchievementListItemBase_2Kmn7,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_GlobalAchievementListItem_316Uo,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementslist_AchievementList_1UXC2,
+.mainmenuapprunning_OverlayAchievements_2Qced .achievementsheader_StatsBlock__gDkI  {
+  border-radius: 20px;
+}
+
+/* Exit Game Dialog */
+.gamepaddialog_GamepadDialogContent_3joNk {
+  border-radius: 20px;
+}
+
+/* Settings */
+.DialogCheckbox_Container,
+.gamepadslider_EditableValue_2DXnl  {
+  border-radius: 20px !important;
```

Comment:
This is the check-boxes container as well as the container for the value of sliders